src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1662,6 +1677,21 @@ private void sendChat(final String key, @Nullable final Object... msg)
         LanguageHandler.sendPlayersMessage(colony.getMessageEntityPlayers(),  this.getColonyJob().getName(), colonyDescription, citizenDescription, requiredItem);
     }
 
+    @Override
+    public boolean hasCapability(net.minecraftforge.common.capabilities.Capability<?> capability, net.minecraft.util.EnumFacing facing)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Unnecessary use of fully qualified name 'net.minecraft.util.EnumFacing' due to existing import 'net.minecraft.util.*' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)